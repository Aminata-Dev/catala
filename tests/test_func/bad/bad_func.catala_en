## Article

```catala
declaration scope S:
  context f content integer depends on integer
  context b content boolean
  context output out content integer

declaration scope R:
  s scope S
  context output r content integer

scope S:
  definition f of x under condition (x >= x) consequence equals x + x
  definition f of x under condition not b consequence equals x * x
  definition b equals false
  definition out equals f of 5

scope R:
  definition s.f of x equals x + x * x
  definition r equals s.out
```

```catala-test {id="R.Interpret"}
catala Interpret -s R
```

```catala-test {id="S.Interpret"}
catala Interpret -s S
```
