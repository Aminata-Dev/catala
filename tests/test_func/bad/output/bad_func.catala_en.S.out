[ERROR] There is a conflict between multiple exceptions for assigning the same variable.

This justification is true:
  --> test_func/bad/bad_func.catala_en
   | 
15 |   definition f of x under condition not b consequence equals x * x
   |                                     ^^^^^
   + Article

This justification is true:
  --> test_func/bad/bad_func.catala_en
   | 
14 |   definition f of x under condition (x >= x) consequence equals x + x
   |                                      ^^^^^^
   + Article
